tests/phpunit.xml

```diff
@@ -5,7 +5,7 @@
 		 timeoutForMediumTests="900"
 		 timeoutForLargeTests="900"
 		>
-	<testsuite name='ownCloud - Mail App Tests'>
+	<testsuite name='Mail App Tests'>
```

Comment:
»Mail app tests«. We only capitalize the first letters of the first word or of app names. ;)